src/Illuminate/Encryption/composer.json

```diff
@@ -16,6 +16,7 @@
     "require": {
         "php": ">=5.5.9",
         "ext-openssl": "*",
+        "ext-mbstring": "*",
```

Comment:
This change is unrelated, and not needed as we already require this extension because we require illuminate/support, which requires it.